readme.md

```diff
@@ -52,47 +62,64 @@ body{
   background-color = "red";
 }
 ```
+```
+Clark Answer: The syntax is incorrect for CSS. Is should be written like this:
 
-> Your answer...
-
+body {
+  background-color: red;
+}
+```
 ## Question 5
 
-**The scenario:** You're starting a new app. You create a local repo and a repo on Github, create a readme file, and put it on Github. Then you run into Juan, who's already done a lot of the work you were planning to do. You want to pull his code down and include it in your repo, and put your combined code up on your Github repo.
+**The scenario:** You're starting a new app. You create a local repo and a repo on Github, create a readme file, and put it on Github. Then you run into Juan, who's already done a lot of the work you
+were planning to do. You want to pull his code down and include it in your repo, and put your combined code up on your Github repo.
 
 All the steps for one way of doing the above have been written below, but in the wrong order. Put them in the correct order.
 
 ```
-$ touch README.md
-$ git push origin master
-$ git push origin master
-$ git remote add origin git@github.com/username/project-repo.git
-$ git remote add juan git@github.com/juan/project-repo.git
-$ git commit -m "initial commit"
 $ cd project-repo
 $ git init project-repo
+$ touch README.md
+$ git remote add origin git@github.com/username/project-repo.git
 $ git add .
+$ git commit -m "initial commit"
+$ git push origin master
+$ git remote add juan git@github.com/juan/project-repo.git
 $ git merge juan/feature
+$ git push origin master
 ```
 
 ## Question 6
 
-Your Rails database has two tables. `students` has the columns `id` and `name`, and `sandwiches` has the columns `id`, `type`, and `student_id`.
+Your Rails database has two tables. `students` has the columns `id` and `name`, and
+`sandwiches` has the columns `id`, `type`, and `student_id`.
 
-Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
+Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named
+Geraldo.
 
 ```rb
-# Your answer...
+class Sandwich < ActiveRecord::Base
+   belongs_to: student
+ end
+ @geraldo = Student.find(name: "Geraldo")
+ @pbj = Sandwich.create(id: 1, type: "pbj", student_id: @geraldo.id)
 
```

Comment:
You have all the code you need here. But we never explicitly set the `id` of a class instance in Rails. So, using what you already wrote, proper could would look like this...

```rb
@geraldo = Student.find(name: "Geraldo")
@pbj = Sandwich.create(type: "pbj", student_id: @geraldo.id)
```

You could also write the code this way...

```rb
@geraldo = Student.find(name: "Geraldo")
@pbj = @geraldo.sandwiches.create(type: "pbj")
```